features/step_definitions/crd.rb

```diff
@@ -9,3 +9,23 @@
     raise "#{crd} not supported by this step"
   end
 end
+
+Given /^the #{QUOTED} #{QUOTED} CR is restored after scenario$/ do |name,crd|
```

Comment:
space between name,crd ?

Reply:
@xingxingxia  CR or CRD which one is better ?

Reply:
CR. Mentioned in https://github.com/openshift/cucushift/pull/7479#pullrequestreview-378194597 .